src/main/java/org/folio/rest/impl/CirculationSettingsAPI.java

```diff
@@ -21,7 +27,10 @@ public void postCirculationSettingsStorageCirculationSettings(String lang,
 
     new CirculationSettingsService(vertxContext, okapiHeaders)
       .create(circulationSettings)
-      .onComplete(asyncResultHandler);
+      .onSuccess(reply -> asyncResultHandler.handle(
+        succeededFuture(respond201WithApplicationJson(circulationSettings, headersFor201()))))
+      .onFailure(errorReply -> asyncResultHandler.handle(
```

Comment:
It is Handler<Throwable>, so it's better to rename it to "throwable -> " or "t -> "

Reply:
done